server/tests/performance/create_test_matrix.py

```diff
@@ -1,44 +0,0 @@
-import anndata
-import argparse
-import random
-import scipy
-import numpy as np
-
-
-def main():
-    parser = argparse.ArgumentParser("A command to generate test h5ad files")
-    parser.add_argument("output", help="Name of the output file")
-    parser.add_argument("nobs", type=int, help="Number of observations (rows)")
-    parser.add_argument("nvar", type=int, help="Number of variables (columns)")
-    parser.add_argument("-n", "--nnz-percent", type=float, default=100, help="percent of non-zeros")
-    parser.add_argument("-c", "--col-shift", action="store_true", help="add a random value to each column")
-    parser.add_argument("--seed", type=int, default=None, help="add a random value to each column")
-
-    args = parser.parse_args()
-    create_test_h5ad(args.output, args.nobs, args.nvar, args.nnz_percent, args.col_shift, args.seed)
-
-
-def create_test_h5ad(outfile, nobs, nvar, nnz_percent=100, apply_col_shift=False, seed=None):
```

Comment:
arguably, should rewrite this for CXG files; it's only used by diffexp unit tests, and for now the repo contains static CXG datasets for the tests, so need for dynamic generation.

Reply:
TODO: util functions to dynamically generate cxgs for tests (cooldown work)